ale/config.py

```diff
@@ -6,7 +6,7 @@
 spice_root = "/data/spice/"
 cassini = '/usgs/cpkgs/isis3/data/cassini/kernels/mk/' # Cassini ISS
 mdis = '/data/spice/mess-e_v_h-spice-6-v1.0/messsp_1000/extras/mk' # Messenger
-mro = '/data/spice/mro-m-spice-6-v1.0/mrosp_1000/extras/mk' # Mars Reconnaissance Orbiter
+mro = '/usgs/cpkgs/isis3/data/mro/kernels/mk' # Mars Reconnaissance Orbiter
```

Comment:
config.py doesn't exists anymore. Pull down the latest changes and update config.yaml.